chatops-lark/pkg/events/handler/root.go

```diff
@@ -193,22 +224,37 @@ func (r *rootHandler) Handle(ctx context.Context, event *larkim.P2MessageReceive
 }
 
 func (r *rootHandler) handleCommand(ctx context.Context, command *Command) (string, error) {
+	cmdLogger := r.logger.With().
+		Str("command", command.Name).
+		Str("sender", command.Sender.Email).
+		Logger()
+
+	cmdLogger.Debug().Msg("Handling command")
+
 	switch command.Name {
 	case "/devbuild":
+		cmdLogger.Debug().Interface("args", command.Args).Msg("Running devbuild command")
 		runCtx := context.WithValue(ctx, ctxKeyLarkSenderEmail, command.Sender.Email)
 		return runCommandDevbuild(runCtx, command.Args)
 	case "/cherry-pick-invite":
+		cmdLogger.Debug().Interface("args", command.Args).Msg("Running cherry-pick-invite command")
 		runCtx := context.WithValue(ctx, ctxKeyGithubToken, r.Config["cherry-pick-invite.github_token"])
 
 		ret, err := runCommandCherryPickInvite(runCtx, command.Args)
 		if err == nil {
 			auditCtx := context.WithValue(ctx, "audit_webhook", r.Config["cherry-pick-invite.audit_webhook"])
-			_ = r.audit(auditCtx, command)
+			if auditErr := r.audit(auditCtx, command); auditErr != nil {
+				cmdLogger.Warn().Err(auditErr).Msg("Failed to audit command")
+			} else {
+				cmdLogger.Debug().Msg("Command audited successfully")
+			}
```

Comment:
```suggestion
			if err := r.audit(auditCtx, command); err != nil {
				cmdLogger.Warn().Err(err).Msg("Failed to audit command")
			} else {
				cmdLogger.Debug().Msg("Command audited successfully")
			}
```

make the err var be shorter.